src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -214,9 +223,20 @@ class Meta:
         }
 
     def create(self, validated_data):
-        return self.Meta.model.objects.generate(
-            validated_data["bronorganisatie"],
-            date.today(),
+        aantal = validated_data.pop("aantal", 1)
```

Comment:
Could you double check if the default is necessary? If it's not, let's remove it, that way the default is declared in one place (in the `aantal` field declared at the top of the serializer)

```suggestion
        aantal = validated_data.pop("aantal")
```